.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://github.com/tanx-libs/precommit-hooks
+    rev: v1.0.1
+    hooks:
+    -   id: private_key_check
```

Comment:
Can you add this in a separate pr?

Reply:
moved the pre commit hook commit to a different PR